cms/static/js/utils/date_utils.js

```diff
@@ -61,7 +61,7 @@ function($, date, TriggerChangeEventOnEnter) {
         // object that corresponds to the datetime.js that they represent. Assume
         // UTC timezone, NOT the timezone of the user's browser.
         var date = null, time = null;
-        if (datepickerInput) {
+        if (datepickerInput.length > 0) {
```

Comment:
Get rid of this check-- you have to at least have a datepicker for this utility method to be useful.